src/DDLQueryBuilder.php

```diff
@@ -173,21 +175,18 @@ public function renameColumn(string $table, string $oldName, string $newName): s
      */
     public function getColumnDefinition(string $table, string $column): string
     {
-        $result = '';
         $sql = $this->schema->getTableSchema($table)?->getCreateSql();
 
         if (empty($sql)) {
-            return '';
+            throw new InvalidArgumentException("Table \"$table\" does not exist.");
```

Comment:
Add information about this exception to phpdoc of this method.

Reply:
`throw` was removed